blenderproc/python/utility/InstallUtility.py

```diff
@@ -200,7 +200,7 @@ def __call__(self, block_num, block_size, total_size):
         elif platform == "darwin":
             blender_run_path = os.path.join(blender_path, "Contents", "MacOS", "Blender")
         elif platform == "win32":
-            blender_run_path = os.path.join(blender_install_path, "blender-windows64", "blender")
+            blender_run_path = os.path.join(blender_install_path, blender_version, "blender")
```

Comment:
This is one possible solution, but I would like to keep the `blender-version` string here, as this will lead to hard to debug bugs, if we change the blender version in the future. Better would be to rename the downloaded `blender-windows64` folder to `blender-version` after install, then this will still work.

Reply:
I'm sorry, I think I cannot follow. Could it be that you are confusing the old code with the new code? 

The hard coded `blender-windows64` is how it was previously named (Blender 2.93 and prior) and this fix changes this to the new naming convention by Blender, which is `blender-{major}.{minor}-windows-x64`, (e.g. `blender-3.0.0.-windows-x64` for Blender 3.0.0). Luckily, this stirng is already stored in `blender_version`, since this is also the name of the parent directory.

Reply:
Yes you are absolutely right! I shouldn't review PRs on my phone :D